controllers/api.articles.comments.controller.js

```diff
@@ -21,3 +21,27 @@ exports.getArticlesComments = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postComment = (req, res, next) => {
+  const { username, body } = req.body;
+  //Validate username and body
+  if (!username || !body) {
+    return next({ status: 400, msg: "Error - username and body are required" });
+  }
+  if (typeof username !== "string" || typeof body !== "string") {
+    return next({ status: 400, msg: "Error - invalid data type" });
+  }
```

Comment:
There's nothing wrong with doing the above - but it is unecessary. If there is no username or body - or if they are the incorrect data type - psql will throw an error of it's own when trying to insert it into the database. 

So instead of these manual checks you can just catch an error in your controller that will be thrown for you